android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -216,11 +216,15 @@
       kmw['executePopupKey'](keyID);
     }
 
-    function executeHardwareKeystroke(code, shift, lstates) {
+    function executeHardwareKeystroke(code, shift, lstates, eventModifiers) {
       var kmw=window['keyman'];
       //window.console.log('executeHardwareKeystroke:('+code+', ' + shift + ', ' + lstates + ');');
       try {
         var r = kmw['executeHardwareKeystroke'](code, shift, lstates);
+        if (r == false) {
```

Comment:
Just checking that this isn't supposed to be `===` (if not, then probably better to remove the variable and say `if(!kmw[...])` because it simplifies the code